nature.md

```diff
@@ -549,7 +545,9 @@ Some pages (e.g in checkout flows) do not have unique page titles at every stage
 <td>
 The meaning or operation of the pages on this product is generally unaffected by focus order. Additionally, in most cases, the focus order of all page content matches the visual order.
 <br><br>
-Third party scripts including advertising scripts, the Disqus commenting widget, or survey or customer services widgets (among others), may inject content where the focus order does not match the visual order. The operation and meaning of these components should be unaffected by the focus order.
+On Nature Outlook pages built with Shorthand, collapsing the main navigation into a hamburger menu may disrupt the focus order, making it harder for keyboard users to access navigation links.
+<br><br>
+Third-party scripts including advertising scripts or survey or customer services widgets (among others), may inject content where the focus order does not match the visual order. The operation and meaning of these components should be unaffected by the focus order.
```

Comment:
Customer services widgets: we might not have these anymore. Check with the FEDs - it used to be that on every page, you'd have a little button on the side near the bottom for opening up a little Freshdesk javascript help widget. I don't see this on Nature now, so it might have been entirely removed.